lib/attributes-to-props.js

```diff
@@ -63,33 +64,13 @@ function cssToJs(style) {
         throw new Error('`cssToJs`: first argument must be a string. ');
     }
 
-    var result = {};
-    // e.g., `color: #f00`
-    var declarations = style.split(';');
-    // css property itemized as key and value
-    var properties;
-    var j;
-    var propertiesLen;
+    var styleObj = {};
 
-    for (var i = 0, declarationsLen = declarations.length; i < declarationsLen; i++) {
-        properties = declarations[i].trim().split(':');
+    parser(style, function(propName, propValue) {
+        styleObj[utilities.camelCase(propName)] = propValue;
```

Comment:
One thing about the style parser is that css comments is included in the iterator. Could we include an `undefined` check here?
```js
if (propName && propValue) {
  styleObj[utilities.camelCase(propName)] = propValue;
}
```